lib/questionnaires/model/item/src/response_model.dart

```diff
@@ -91,7 +92,7 @@ class ResponseModel {
     return _cachedAnswerModels.values.any((am) => !am.isUnanswered);
   }
 
-  final _cachedAnswerModels = <int, AnswerModel>{};
+  final Map<int, AnswerModel> _cachedAnswerModels = <int, AnswerModel>{};
```

Comment:
doesn't Dart derive that properly?

Reply:
Yes, but I tend to err on the side of strongly typing my initializers. Both are equivalent. The reason why this change occurred in the first place was because I was testing what would happen if this was a `List<AnswerModel>` instead. Currently, your `int` seems to match what would have otherwise been your list index...so I'm not really sure why this began as a map